docs/content/en/docs/tasks/troubleshoot/troubleshooting.md

```diff
@@ -138,20 +125,148 @@ If you don't have any other docker containers running you may want to run `docke
+```
+This is likely a [Memory or disk resource problem]({{< relref "#memory-or-disk-resource-problem" >}}).
+
+### Generic cluster unavailable
+Troubleshoot more by inspecting bootstrap cluster or workload cluster (depending on the stage of failure) using kubectl commands. 
+```
+kubectl get pods -A --kubeconfig=<kubeconfig>
+kubectl get nodes -A --kubeconfig=<kubeconfig>
+kubectl get logs <podname> -n <namespace> --kubeconfig=<kubeconfig>
+....
+```
+Capv troubleshooting guide: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/troubleshooting.md#debugging-issues
+
+## Bare Metal troubleshooting
+
+### Bootstrap cluster fails to come up
+
+```
+Error: creating bootstrap cluster: executing create cluster: ERROR: failed to create cluster: node(s) already exist for a cluster with the name \"cluster-name\"
+, try rerunning with —force-cleanup to force delete previously created bootstrap cluster
+```
+
+The cluster already exists, so you can delete the old one:
+
+```bash
+kind delete cluster --name cluster-name
+```
+
+### Creating new workload cluster hangs or fails
+
+Cluster creation appears to be hung waiting for the Control Plane to be ready.
+If the CLI is hung on this message for over 30 mins, something likely failed during the OS provisioning:
+
+```
+Waiting for Control Plane to be ready
+```
+
+Or if cluster creation times out on this step and fails with the following messages:
+
+```
+Support bundle archive created {"path": "support-bundle-2022-06-28T00_41_24.tar.gz"}
+Analyzing support bundle {"bundle": "CLUSTER_NAME/generated/bootstrap-cluster-2022-06-28T00:41:24Z-bundle.yaml", "archive": "support-bundle-2022-06-28T00_41_24.tar.gz"}
+Analysis output generated {"path": "CLUSTER_NAME/generated/bootstrap-cluster-2022-06-28T00:43:40Z-analysis.yaml"}
+collecting workload cluster diagnostics
+Error: waiting for workload cluster control plane to be ready: executing wait: error: timed out waiting for the condition on clusters/CLUSTER_NAME
+```
+In either of those cases, the following steps can help you determine the problem:
+
+1. Export the kind cluster’s kubeconfig file:
+
+    ```bash
+    export KUBECONFIG=${PWD}/${CLUSTER_NAME}/generated/${CLUSTER_NAME}.kind.kubeconfig
+    ```
+1. If you have provided BMC information:
+
+    * Check all of the machines that the EKS Anywhere CLI has picked up from the pool of hardware in the CSV file:
+
+        ```bash
+        kubectl get machines -A
+        ```
+    * Check if those nodes are powered on. If any of those nodes are not powered on after a while then it could be possible that BMC credentials are invalid. You can verify it by checking the logs:
+
+        ```bash
+        kubectl get bmt -n eksa-system
```

Comment:
should this be bmc? instead of bmt. @pokearu

Reply:
`kubectl get bm` is also an option to check if the credentials or connectivity is at fault.
But we can look at bmt (BMCTask) as the task would have failed and the failure message in the task conditions would specify if it's a connection error or any other issue.

Reply:
ah so it is bmt. Thanks for confirming

Reply:
It seems no change is needed here. Let me know if I misinterpreted.